test/replication-luatest/gh_8121_memtx_mvcc_replication_stream_test.lua

```diff
@@ -0,0 +1,61 @@
+local t = require('luatest')
+local replica_set = require('luatest.replica_set')
+local server = require('luatest.server')
+
+local g = t.group(nil, t.helpers.matrix({engine = {'memtx', 'vinyl'}}))
+
+g.before_all(function(cg)
+    cg.replica_set = replica_set:new()
+    cg.master = cg.replica_set:build_and_add_server({alias = 'master'})
+    cg.replica = cg.replica_set:build_and_add_server(
+            {
+                alias = 'replica',
+                box_cfg = {
+                    replication = server.build_listen_uri('master'),
+                    memtx_use_mvcc_engine = true,
+                    txn_isolation = 'read-confirmed',
+                }
+            })
```

Comment:
Don't forget to lower `replication_timeout`. We usually set 0.1, and it speeds the tests up quite a bit. For example for this test the time went down from ~ 3.4 seconds to ~ 1.6 on my machine.
```diff
diff --git a/test/replication-luatest/gh_8121_memtx_mvcc_replication_stream_test.lua b/test/replication-luatest/gh_8121_memtx_mvcc_replication_stream_test.lua
index 1157a287c..7debcda00 100644
--- a/test/replication-luatest/gh_8121_memtx_mvcc_replication_stream_test.lua
+++ b/test/replication-luatest/gh_8121_memtx_mvcc_replication_stream_test.lua
@@ -6,12 +6,19 @@ local g = t.group(nil, t.helpers.matrix({eng = {'memtx', 'vinyl'}}))
 
 g.before_all(function(cg)
     cg.replica_set = replica_set:new()
-    cg.master = cg.replica_set:build_and_add_server({alias = 'master'})
+    cg.master = cg.replica_set:build_and_add_server(
+            {
+                alias = 'master',
+                box_cfg = {
+                    replication_timeout = 0.1,
+                },
+            })
     cg.replica = cg.replica_set:build_and_add_server(
             {
                 alias = 'replica',
                 box_cfg = {
                     replication = server.build_listen_uri('master'),
+                    replication_timeout = 0.1,
                     memtx_use_mvcc_engine = true,
                     txn_isolation = 'read-confirmed',
                 }
```

Reply:
Ok, fixed. Why isn't this the default value then?